app/models/vacancy.rb

```diff
@@ -358,6 +358,7 @@ def slug_candidates
       :job_title,
       %i[job_title school_or_school_group_name],
       %i[job_title location],
+      :id
```

Comment:
Is this because you need to set the slug without a job title being specified? I think you can wait to set the slug until the job_specification page

Reply:
It is because of that.

Reply:
Setting the slug as the id doesn't make much sense to me, we may as well not set a slug at that point

Reply:
It's because the slug can't be null, at the moment. The slug is refreshed whenever the vacancy is updated, so as soon as we can have a meaningful text, it will use that. Worth making `slug` nullable?

Reply:
Yeh I think that makes more sense